src/http/errors.ts

```diff
@@ -44,6 +44,22 @@ function extractErrorProps(res: Any): ErrorProps {
     return props
   }
 
+  // Mutation errors (specifically)
+  if (isMutationError(body)) {
+    const allItems = body.error.items || []
+    const items = allItems
+      .slice(0, 5)
```

Comment:
If we want to change the number of errors shown by default, it's easy to miss one of these `5`'s and get an inaccurate message. Put it in a constant instead?

Reply:
Great point! Have changed to a constant.